src/Symfony/Component/Messenger/Middleware/UnwrapHandlerExceptionMiddleware.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Middleware;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Exception\HandlerFailedException;
+use Symfony\Component\Messenger\Exception\LogicException;
+
+class UnwrapHandlerExceptionMiddleware implements MiddlewareInterface
+{
+    public function handle(Envelope $envelope, StackInterface $stack): Envelope
+    {
+        try {
+            return $stack->next()->handle($envelope, $stack);
+        } catch (HandlerFailedException $exception) {
+            $wrappedExceptions = $exception->getWrappedExceptions();
+
+            if (1 !== \count($wrappedExceptions)) {
+                throw new LogicException(sprintf('"%s" can only unwrap a single exception, but got %d.', __CLASS__, \count($wrappedExceptions)));
```

Comment:
```suggestion
                throw new LogicException(sprintf('%s can only unwrap a single exception, but got %d.', __CLASS__, \count($wrappedExceptions)));
```

I think it is more readable

Reply:
fabbot doesn't like it 😅

Reply:
I think we can ignore it, @nicolas-grekas?

Reply:
indeed